packages/graphql-transformer-common/src/ResourceConstants.ts

```diff
@@ -51,6 +65,16 @@ export class ResourceConstants {
 
         // Auth
         AuthCognitoUserPoolId: 'AuthCognitoUserPoolId',
+        AuthCognitoUserPoolName: 'AuthCognitoUserPoolName',
+        AuthCognitoUserPoolMobileClientName: 'AuthCognitoUserPoolMobileClientName',
+        AuthCognitoUserPoolJSClientName: 'AuthCognitoUserPoolJSClientName',
+        AuthCognitoUserPoolRefreshTokenValidity: 'AuthCognitoUserPoolRefreshTokenValidity',
+
+        // Aurora Serverless
+        rdsRegion: 'rdsRegion',
```

Comment:
Can we move these constants into the new package?

Reply:
done